setup.py

```diff
@@ -145,6 +145,7 @@
         ],
         'invenio_base.apps': [
             'invenio_accounts = invenio_accounts:InvenioAccounts',
```

Comment:
You should remove this line, since you're calling ``super(InvenioAccountsUI, self).init_app(..)`` above